includes/class-piklist-form.php

```diff
@@ -2149,13 +2149,13 @@ public static function validate_field_parameter($parameter, $value)
     {
       case 'capability':
 
-        return piklist_user::current_user_can($value);
+        return $value == 'none' ? true : piklist_user::current_user_can($value);
 
       break;
 
       case 'role':
 
-        return piklist_user::current_user_role($value);
+        return $value == 'none' ? true : piklist_user::current_user_role($value);
```

Comment:
Let's use triple equal checks as often as possible, and in light of my last comment this could instead be:
```php
null === $value ? true : piklist_user::current_user_role($value)
```
Same for the above line.

Reply:
@JasonTheAdams `null` doesn't work. These references would have to change as well:https://github.com/piklist/piklist/blob/develop/includes/class-piklist-form.php#L3199

https://github.com/piklist/piklist/blob/develop/includes/class-piklist-form.php#L3280

https://github.com/piklist/piklist/blob/develop/includes/class-piklist-form.php#L3467

https://github.com/piklist/piklist/blob/develop/includes/class-piklist-form.php#L3730